src/content/doc-surrealdb/deployment/surreal cloud.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 1
+sidebar_label: Deploy on Surreal Cloud (Beta)
+title: Deploy on Surreal Cloud (Beta) | Deployment
+description: This tutorial guides users through the process of deploying SurrealDB using Surreal Cloud.
```

Comment:
'Users' feels slightly cold to me (like it's specifically not addressed to the reader), how about something like "This tutorial provides a guide to the process..."?